website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,33 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Select
+
+Multi-select enables users to select multiple rows from a dataset, significantly enhancing the efficiency of batch operations like delete, move, or update. This functionality is important for effective data management, especially in large datasets.
+
+!!! Info 
+
+Avoid using indeterminate states in individual rows. It's more effective at the global selection checkbox to reduce confusion.
```

Comment:
I think you can be more pointed here and essentially say that indeterminate state isn't supported in individual rows. Think back to what we discussed about parent/child relationships here. The rows (in the majority of scenarios) don't have the concept of children, therefore they can only be considered on/off, checked/un-checked.